packages/gatsby-theme-aio/src/components/Hero/index.js

```diff
@@ -159,6 +162,7 @@ const Hero = ({
     const pagesWithRootFix = rootFixPages(pages);
     const selectedTopPage = findSelectedTopPage(pathWithRootFix, pagesWithRootFix);
     const selectedTopPageMenu = findSelectedTopPageMenu(pathWithRootFix, selectedTopPage);
+    showBreadcrumbNav = getCleanBoolean(showBreadcrumbNav); // this will cast it to a boolean if its a 0 1 or true false string
```

Comment:
I think `getCleanBoolean` can be removed. We should only support `true` or `false`.

Reply:
It does not error when set as a string.  I think its safer this way.